content/docs/user-guide/contributing/blog.md

```diff
@@ -1,7 +1,7 @@
 # Writing Blog Posts
 
-To create a new post, you need to create a PR to the DVC website
-[project](https://github.com/iterative/dvc.org).
+To create a new post, you need to create a PR to the DVC website super mega
+ultra great [project](https://github.com/iterative/dvc.org).
```

Comment:
OK hold on so https://github.com/iterative/dvc.org/pull/2775/files supposedly fixed this ☝️ (which I need to undo) but it's still not fixed according to [the check](https://app.circleci.com/pipelines/github/iterative/dvc.org/10439/workflows/08bb5f08-74d4-4487-9518-7ac3ce117250/jobs/10498). The line looks technically good to me... ⏳